docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Gatsby Docs Translation Guide
+---
+
+Since the [RFC on localization](https://github.com/gatsbyjs/rfcs/pull/42) for Gatsby, there is now an ongoing effort to translate the content on Gatsbyjs.org into multiple languages. For members of the community around the world building Gatsby sites and learning about web development, having docs and learning materials in your own language provides a lot of value. There's also a great opportunity for folks like you to contribute to the translations of the Gatsby docs.
+
+If you're a native speaker of a language other than English and you're interested in helping translate gatsbyjs.org, here is some information for you to know:
+
+## The general process
+
+Each translation has its own repository in the [gatsbyjs](https://github.com/gatsbyjs/) organization on GitHub, with two or three designated codeowners to review and approve changes. This doc will outline how to request a new language repository with its own team, what decisions each team can make, and suggestions for how you can work together.
+
+The Gatsby learning team is in charge of determining priorities for which docs should be translated. Refer to the [i18n page spreadsheet](https://docs.google.com/spreadsheets/d/1u2amGnqFLKxJuL5h9UrDblUueFgg0EBt7xbau4n8iTM/edit) to get the most up-to-date priority list.
+
+> ⚠️ Note: All contributors are expected to follow the [Gatsby Code of Conduct](/contributing/code-of-conduct/) and work professionally with fellow contributors. For issues with conduct, if you are unable to work things out amicably amongst yourselves (perhaps after filing a public issue), you can contact the Gatsby team at [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) with details about the situation.
```

Comment:
Great idea! I added it to the top of the priorities spreadsheet.

Reply:
The email `conduct@gatsbyjs.com` doesn’t actually work, I tried to write to this address, but received an error about sending. I also wrote in DM on Twitter, but they also did not reply.

It’s unfortunate that the Gatsby team doesn’t care what kind of problems are happening to their community!

Reply:
Hi @lex111, the conduct alias does work, and we do care -- our sincere apologies for taking a while to get back to you. Our request has been that situations be attempted to remedy amongst yourselves in the open through GitHub issues first, and that is what you were asked to do originally by Nat. However, we recognize that you were missing write access and that likely contributed to the feeling of being ignored. As early pioneers of the Gatsby translation project, you've helped us iron out some of the issues with repos, process, and governance around translations of which we'll do our best to represent and maintain in this doc. It looks like you've since [opened an issue](https://github.com/gatsbyjs/gatsby-ru/issues/19) in the Gatsby-ru project that seemed to have a productive outcome, so thanks for doing that (and keep opening issues as needed)!